src-symfony-compatibility/v6/Style/DrushStyle.php

```diff
@@ -23,7 +23,7 @@ public function confirm(string $question, bool $default = true): bool
         return parent::confirm($question, $default);
     }
 
-    public function choice(string $question, array $choices, mixed $default = null): mixed
+    public function choice(string $question, array $choices, mixed $default = null, bool $multiSelect = false): mixed
```

Comment:
We need to pass along $multiSelect in the parent::choice call a few lines down.

Reply:
:facepalm: fixed - to clarify this is only when we get to symfony/console 6.2

Reply:
its safe to merge this now right? its compatible with symfony 4?

Reply:
I think you have a different DrushStyle for symfony4 in the src-symfony-compatibility/v4 directory.

In terms of will it break on symfony/console: 6.1 - doesn't seem like it https://3v4l.org/ZXXWg